docs/theme.rst

```diff
@@ -8,6 +8,11 @@ The :doc:`theme/default` has a templating engine (previously called "themes" in
 Liquid Prompt version 1), that allows for custom prompt ordering in the default
 theme.
 
+If you just want to change icons and/or colors, you can just rely on "presets",
+which are just regular configuration files.
+Presets can generally be combined, and some themes may use
+—or be compatible with— different presets. See :doc:`config`.
```

Comment:
Is this a French thing? English generally uses commas to break sentences like this:
> and some themes may use, or be compatible with, different presets.

or parens:
> and some themes may use (or be compatible with) different presets.

The Em dash is still in use, but spaces surrounding them should be removed:
> and some themes may use—or be compatible with—different presets.

I'm not sure how to remove the space when it falls on a line break.

Reply:
In both french and english (regardless of their national variants) the em dash may be used when a comma is too weak and parentheses too strong (which I felt was the case here).

As for typography, french use them opened (space dash thin-space word), while in english it is either closed (no space) or opened (space dash hair-space word). For instance, the AP style book sets it open with the smallest ("hair") space, which I render as semi-open, probably as a compromise with my inner french and the necessity of correct line breaks (which are usually not good with the closed version, whether the open one allows you to use non-breaking space).

As usual, Wikipedia has an article about that question: https://en.wikipedia.org/wiki/Dash#Em_dash

All in all, if it's too pompous in american english, we may very well get rid of it (and use parentheses) without me getting the slightest upset.

Reply:
Nice summary. I probably prefer parentheses because of the logic/programming connotations. But you are correct, there is nothing wrong with the Em dash. I'm fine leaving it.